guacamole/src/licenses/LICENSE

```diff
@@ -605,6 +605,36 @@ licenses; we recommend you read them, as their terms may differ from the
 terms above.
 
 
+JSTZ (http://webjars.org/)
+--------------------------
+
+    Version: 1.0.10
+    From: 'WebJars' (http://webjars.org/)
```

Comment:
"From" here needs to be the copyright holder. Based on the license below, it looks like the correct value would be `'Jon Nylander'`.

If the individual/organization holding the copyright has a website, a GitHub account, etc., that's what has been used for the URL following the name.